### PR TITLE
Documentation/manual update

### PR DESCRIPTION
**Pull request by KochTobi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,10 +15,14 @@ A Liferay portlet to update the sample-tracking information of selected samples.
...
 ## How to Install
-1. clone repository
-2. install maven
-3. run `mvn clean package`
-
+### Prerequisites
+1. A [Properly Configured Liferay Portal](https://github.com/qbicsoftware/qbic-setup-doc/blob/main/application_servers/liferay7.md) <!-- TODO: Where to link?-->
+2. Access to a running instance of the [sample-tracking service](https://github.com/qbicsoftware/sample-tracking-service)
+   
+3. An properly configured [openBIS instance](https://openbis.ch/) <!-- TODO where to link? -->
```
Comment (jenniferboedker) [inline README.md:22]: what about the todos? Can they be solved

Comment (KochTobi) [reply README.md:22]: nope there is no documentation as of now. The link to openbis partly fulfills the TODO for now but must be tackled again later on.

File: README.md [hunk 2]
```diff
@@ -15,10 +15,14 @@ A Liferay portlet to update the sample-tracking information of selected samples.
...
+4. A [configuration file](https://github.com/qbicsoftware/portal-utils-lib#how-portlets-are-configured) providing information about the location of the sample-tracking service as well as the database and the openBIS instance
+### Installation
+Either run it locally or deploy it on your liferay instance. To run it locally you can execute ``mvn clean jetty:run``. To deploy it on your liferay instance you can create the required ``.war`` file by executing ``mvn clean package``
 ## How to Use
```
Comment (jenniferboedker) [inline README.md:26]: when writing everything in upper case (as done for most of the lower captions) you should do it everywhere

File: README.md [hunk 3]
```diff
@@ -15,10 +15,14 @@ A Liferay portlet to update the sample-tracking information of selected samples.
 
 
 ## How to Install
-1. clone repository
-2. install maven
-3. run `mvn clean package`
-
+### Prerequisites
+1. A [Properly Configured Liferay Portal](https://github.com/qbicsoftware/qbic-setup-doc/blob/main/application_servers/liferay7.md) <!-- TODO: Where to link?-->
```
Comment (KochTobi) [inline README.md:19]:
```suggestion
1. A [Properly Configured Liferay Portal](https://liferay.dev/blogs/-/blogs/liferay-6-2-ce-ga6-now-available) <!-- TODO: Where to link?-->
```

File: README.md (hunk 1 above)
Comment (jenniferboedker) [reply README.md:22]: So you will add the link as discussed in the daily?

Comment (jenniferboedker) [inline README.md:22]:
```suggestion
3. A properly configured [openBIS instance](https://openbis.ch/) <!-- TODO where to link? -->
```

File: README.md [hunk 4]
```diff
@@ -35,68 +39,71 @@ This guide intends to showcase the features of the sample-tracking-update-portle
 ### Select Samples
 
 The sample-tracking-update-portlet currently offers two methods for adding the samples to update.  
-This can be achieved by either uploading a *.csv file containing the Sample Ids in the first column 
-or by providing the Sample IDs directly.   
+This can be achieved by either uploading a *.csv file containing the sample identifiers in the first column 
+or by providing sample identifiers directly.
```
Comment (jenniferboedker) [inline README.md:43]:
```suggestion
or by providing them directly.   
```

File: README.md [hunk 5]
```diff
@@ -35,68 +39,71 @@ This guide intends to showcase the features of the sample-tracking-update-portle
...
-If the Sample Ids in the csv file are valid, 
+A template file can be found at [``readme-docs/templates/example.csv``](readme-docs/templates/example.csv).
+
+If the sample identifiers in the csv file are valid, and if they are not already in the list of loaded samples, 
 the samples and their associated information will appear in the sample grid.
 ![add](readme-docs/screenshots/AddSampleByFileResult.png)
   
-Subsequent Sample IDs from either a *.csv file or by direct input will also be added to the "Samples to update" grid. 
-![add](readme-docs/screenshots/AddSampleByFileAgain.png)
+Subsequent sample identifiers from either a *.csv file or by direct input will also be added to the "Samples to update" grid.
```
Comment (jenniferboedker) [inline README.md:68]:
```suggestion
Subsequent sample identifiers from either a csv file or by direct input will also be added to the "Samples to update" grid. 
```
just to be consistent

**Review by jenniferboedker — APPROVED** [review]
LGTM

**Pull request merged**